R/DBI-pool.R

```diff
@@ -47,8 +48,39 @@ setMethod("onDestroy", "DBIConnection", function(object) {
 #' @export
 #' @rdname object
 setMethod("onValidate", "DBIConnection", function(object) {
-  check <- dbGetQuery(object, "SELECT 1")
-  df <- data.frame(1)
-  names(df) <- "1"
-  (check == df)[1,]
+  pool <- attr(object, "..metadata", exact = TRUE)$..pool
```

Comment:
You don't have to change this, but, you don't need to prepend the members of `..metadata` with `..`--just prepending the `..metadata` attr name is enough. And in retrospect perhaps `pool_` would've been a better prefix. (I'm assuming I suggested the `..`)